code/style.css

```diff
@@ -1,4 +1,261 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*{
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+	
+	}
+	body{
+	  background-image: url("https://www.sciencenewsforstudents.org/wp-content/uploads/2019/11/iStock_000019745738Medium_0-772x579.jpg");
+	  background-attachment: fixed;
+	  background-size:100% 100%;
+	  min-height: 100vh;
+	}
+	h1{
+		color: whitesmoke;
+		text-shadow: 1px 1px black;
+	
+	}
+	p {
+		justify-content: center;
+		text-align: center;
+		text-shadow: 1px 1px black;
+		color: white
+	
+	}
+	.flexbox-text p{
+		font-size: 20px;
+	}
+	/* Navbar */
+	nav {
+		z-index: 10;
+		left: 0;
+		right: 0;
+		top: 0;
+		font-family: url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:ital@0;1&display=swap');
+		padding: 0 5%;
+		height: 100px;
+		background: linear-gradient(222deg, #191b1b, #736bc7, #3a09b5) fixed;
+	}
+	
+	.logo {
+		float: left;
+		width: 40%;
+		height: 100%;
+		display: flex;
+		align-items: center;
+		font-size: 20px;
+		color: #fff;
+	}
+	.logo {
+		animation: right 10s ease infinite;
+		gap: 2rem;
+		width:8rem;
+	}
+	nav .links {
+		float: right;
+		padding: 0;
+		margin: 0;
+		width: 60%;
+		height: 100%;
+		display: flex;
+		justify-content: space-around;
+		align-items: center;
+	}
+	nav .links li {
+		list-style: none;
+	}
+	
+	nav .links a {
+		display: block;
+		padding: 10px;
+		font-size: 16px;
+		font-weight: bold;
+		color: #fff;
+		text-decoration: none;
+	}
+
+	#nav-toggle {
+		position: absolute;
+		top: -100px;
+	}
+	nav .icon-burger {
+		display: none;
+		position: absolute;
+		right: 50px;
+		top: 50px;
+		transform: translateY(-50%);
+	}
+	nav .icon-burger .line {
+		width: 30px;
+		height: 5px;
+		background-color: #fff;
+		margin: 5px;
+		border-radius: 3px;
+		transition: all .3s ease-in-out;
+	}
+	/* My flexbox big news*/
+	
+	.flexbox-container {
+		flex-direction: row;
+		font-size: 30px;
+		text-align: center;
+		 
+	}
+	.flexbox-img{
+		padding: 15px;
+		
+	}
+		/* My Grid smaller news  */
+	.grid-contanier, .flexbox-container a{
+		text-decoration: none;
+	}
+	.grid-contanier a{
+		text-decoration: none;
```

Comment:
You could emerge this with the section above since the only element is text-decoration: none;